tests/models/test_regression.py

```diff
@@ -61,12 +61,9 @@ def test_train(self, tmp_path, capsys, use_pred_months, experiment, monthly_agg)
         x_add2, _, _ = _make_dataset(size=(5, 5), const=True, variable_name='temp')
         x = xr.merge([x, x_add1, x_add2])
 
-        norm_dict = {'VHI': {'mean': np.zeros((1, x.to_array().values.shape[1])),
-                             'std': np.ones((1, x.to_array().values.shape[1]))},
-                     'precip': {'mean': np.zeros((1, x.to_array().values.shape[1])),
-                                'std': np.ones((1, x.to_array().values.shape[1]))},
-                     'temp': {'mean': np.zeros((1, x.to_array().values.shape[1])),
-                              'std': np.ones((1, x.to_array().values.shape[1]))}}
+        norm_dict = {'VHI': {'mean': 0, 'std': 1},
```

Comment:
what happens to the other variables?

Reply:
oops! this is a mistake.

Since they are not in the norm dict, they wouldn't be normalized, but I'll add them back to the test

Reply:
What other variables do you mean?

Reply:
whereas before, we had a vector of length 12 (for 12 timesteps) and we divided it by 12 unique values, now we divide all 12 values by the same value.

Reply:
as in `temp` etc.

Reply:
they are still there (lines 65, 66)